java/fury-core/src/main/java/org/apache/fury/serializer/NonexistentClassSerializers.java

```diff
@@ -226,14 +227,21 @@ public Object read(MemoryBuffer buffer) {
 
   public static final class NonexistentEnumClassSerializer extends Serializer {
     private final NonexistentEnum[] enumConstants;
+    private final MetaStringResolver metaStringResolver;
 
     public NonexistentEnumClassSerializer(Fury fury) {
       super(fury, NonexistentEnum.class);
+      metaStringResolver = fury.getMetaStringResolver();
       enumConstants = NonexistentEnum.class.getEnumConstants();
     }
 
     @Override
     public Object read(MemoryBuffer buffer) {
+      if (fury.getConfig().serializeEnumByName()) {
+        metaStringResolver.readMetaStringBytes(buffer);
```

Comment:
wondering, do i need to read the buffer first to make sure the buffer ready for next consumption?

Reply:
I think current implementation is fine.

Reply:
ok, should i cancel and close this PR then?

Reply:
I mean your implementation here is right, no need to change